.github/workflows/x-keycloak-operator.yml

```diff
@@ -146,3 +146,87 @@ jobs:
       - name: Show Container tags
         run: |
           echo "${{ steps.meta.outputs.tags }}" >> $GITHUB_STEP_SUMMARY
+
```

Comment:
This should be split into a separate workflow (something like `x-keycloak-operator-hub-publish`). That will make it easier to maintain and also easier to recover from a failure.

Seeing as we only want to publish releases and not nightlies, this should be only added to the release workflow.

Reply:
+1